### DISCUSSION TITLE
No Use Of Google Cloud APIS Now.....

### DISCUSSION THREAD

**Discussion by shivamguys** [shivamguys/irctc-cypress-automation#35] [Announcements]
As many users were unable to set up Google Cloud API credentials stuff as well as card concerns also related to billing, So this new release has nothing to do with the Google Cloud APIs and is now much easier. 

You Can Also now Fork this repo, modify passenger data, and then run your tatkal using the Actions Tab and fill in your IRCTC Username, Password, and UPI to Make payments.

![image](https://github.com/shivamguys/irctc-cypress-automation/assets/25263989/b1ce22ca-9c0d-4b7e-a483-908614e20cca)

<hr /><em>This discussion was created from the release <a href='https://github.com/shivamguys/irctc-cypress-automation/releases/tag/3.0.1'>No Use Of Google Cloud APIS Now.....</a>.</em>

**Comment by oberoy-ravi** [comment]
hey there, kindly do check the LinkedIn.

**Comment by ravioutlook58-cloud** [comment]
Hi
